.eslintrc

```diff
@@ -0,0 +1,25 @@
+{
+  "env": {
+    "mocha": true,
+    "node": true,
+    "es6": true
```

Comment:
typescript? lol

Reply:
this might have been c/p from CG :]

Reply:
Also, not sure if you are referring to es6 with the comment but that's required if using es6 global variables